### PR TITLE
deverification copy changes

### PR DESCRIPTION
**Pull request by divine-comedian**
deverification copy changes and remove givpower rank modal for vouched projects

## Summary by CodeRabbit

- **New Features**
	- Updated labels and descriptions for the DeVouch feature across multiple languages to enhance clarity and user guidance.
	- Introduced new messaging for GIVbacks eligibility, including dynamic links for staking information.
	- Added tooltips and clearer descriptions in the donation section to improve user experience.

- **Bug Fixes**
	- Corrected navigation and action descriptions to better reflect user actions related to DeVouch and GIVbacks.

- **Documentation**
	- Enhanced clarity in user interface messages regarding project verification and GIVbacks processes.

### CODE REVIEW

File: src/components/views/project/projectGIVPower/index.tsx [hunk 1]
```diff
@@ -28,7 +28,9 @@ const ProjectGIVPowerIndex: FC<IProjectGIVPowerIndexProps> = () => {
 	const { isBoostingsLoading, boostersData, projectData } =
 		useProjectContext();
 	const hasGivPower = boostersData ? boostersData.totalCount > 0 : false;
-
+	const isVerified = projectData?.verified;
+	const isGivbackEligible = projectData?.isGivbackEligible;
```
Comment (RamRamez) [inline src/components/views/project/projectGIVPower/index.tsx:32]: I believe you can safely remove these new variables.

**Review by RamRamez — CHANGES_REQUESTED** [review]
Great! Thanks @divine-comedian 
Just two minor fixes.

**Comment by divine-comedian** [comment]
Thanks @RamRamez ready for another review

**Comment by divine-comedian** [comment]
FYI @MohammadPCh we should merge this BEFORE doing the release

**Review by RamRamez — APPROVED** [review]
Thanks @divine-comedian

**Pull request merged**